src/test/java/com/epam/training/array/ArrayLogicTest.java

```diff
@@ -1,7 +1,71 @@
 package com.epam.training.array;
 
+import com.epam.training.array.impl.ArrayLogic;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.File;
+
 public class ArrayLogicTest {
+    private static final ArrayLogic arrayLogic = new ArrayLogic();
+
+    @Test
+    public void testExists() {
+        Array array = new Array(new int[]{5, 7, 2, 9, 8, 10, 4});
```

Comment:
@AIniuseva Could you divide your tests into three blocks `given`, `when` and `then`. For example:

> // given
> Array array = new Array(new int[]{5, 7, 2, 9, 8, 10, 4});

> // when
> boolean isExist = arrayLogic.exists(array, 10);

> // then
> Assert.assertTrue(isExist);

Reply:
@NikitaUstsiushenka Done. Please, see the changes from here: https://github.com/AIniuseva/array-task/pull/1/files#diff-79f0af9a46c87d1b48827dd8f3e9f108ac8be6bb0df2689157197e8d6593b1b1R9